tests/acceptance/features/bootstrap/WebUIUsersContext.php

```diff
@@ -149,6 +158,32 @@ public function theAdminCreatesAUserUsingTheWebUI(
 		}
 	}
 
+	/**
+	 * @When the administrator attempts to create these users then the notifications should be as listed
+	 *
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 */
+	public function theAdministratorAttemptsToCreateAUser(TableNode $table) {
+		foreach ($table->getHash() as $row) {
+			$user = $row['user'];
+			$this->theAdminCreatesAUserUsingTheWebUI(
```

Comment:
There is already a function for administrator to attempt creating the user. I think it would be better to call that function.